salt/modules/win_service.py

```diff
@@ -443,6 +443,123 @@ def create(name,
+          - boot: Device driver that is loaded by the boot loader
+          - system: Device driver that is started during kernel initialization
+          - auto: Service that automatically starts
+          - demand (default): Service must be started manually
+          - disabled: Service cannot be started
+          - delayed-auto: Service starts automatically after other auto-services
+            start
+
+        :param str error: Specifies the severity of the error if the service
+        fails to start. Acceptable values are:
+          - normal (default): Error is logged and a message box is displayed
+          - severe: Error is logged and computer attempts a restart with last
+            known good configuration
+          - critical: Error is logged, computer attempts to restart with last
+            known good configuration, system halts on failure
+          - ignore: Error is logged and startup continues, no notification is
+            given to the user
+
+        :param str group: Specifies the name of the group of which this service
+        is a member. The list of groups is stored in the registry, in the
+        HKLM\System\CurrentControlSet\Control\ServiceGroupOrder subkey. The
+        default is null.
+
+        :param str tag: Specifies whether or not to obtain a TagID from the
+        CreateService call. For boot-start and system-start drivers only.
+        Acceptable values are:
+          - yes/no
+
+        :param str depend: Specifies the names of services or groups that must
+        start before this service. The names are separated by forward slashes.
+
+        :param str obj: Specifies th name of an account in which a service will
+        run or specifies a name of the Windows driver object in which the driver
+        will run. Default is LocalSystem
+
+        :param str password: Specifies a password. Required if other than
+        LocalSystem account is used.
+
+        :return: True if successful, False if not
+        :rtype: bool
+
+        CLI Example:
+
+        .. code-block:: bash
+
+            salt '*' service.config <service name> <path to exe> display_name='<display name>'
+        '''
+
+        cmd = ['sc', 'config', name]
+        if bin_path is not None:
+            cmd.extend(['binpath=', bin_path])
+        if type is not None:
+            cmd.extend(['type=', type])
+        if start is not None:
+            cmd.extend(['start=', start])
+        if error is not None:
+            cmd.extend(['error=', error])
+        if display_name is not None:
+            cmd.extend(['DisplayName=', display_name])
+        if group is not None:
+            cmd.extend(['group=', group])
+        if depend is not None:
+            cmd.extend(['depend=', depend])
+        if obj is not None:
+            cmd.extend(['obj=', obj])
+        if password is not None:
+            cmd.extend(['password=', password])
+
+        return not __salt__['cmd.retcode'](cmd, python_shell=False)
```

Comment:
Interesting. We should change the documentation. We compile lists for cmd.run in many places in the codebase these days. Were that to change, *major* modules would break.